cmake/modules/SwiftBuildConfig.cmake.in

```diff
@@ -0,0 +1,3 @@
+if(NOT TARGET SwiftBuild)
```

Comment:
What is the target name in the generated SwiftBuildExports file? I believe the namespace means it should be a `SwiftBuild::SwiftBuild` target, no?

Reply:
yeah, I expect I forgot to update this when I added the namespace

Reply:
fixed